twine/settings.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         client_cert: Optional[str] = None,
         repository_name: str = "pypi",
         repository_url: Optional[str] = None,
-        verbose: bool = False,
+        verbose: int = 0,
```

Comment:
This looks like an API change to a public class, which we've avoided in the time that I've been contributing to Twine. Now, it might be fine, thanks to duck typing, but it makes me wonder:

- Do we actually need multiple verbosity levels and the complexity that brings? IE, can we leave it as a boolean in this PR, and iterate towards multiple if/when necessary?

- If we do want multiple levels right away, does this imply the next version is Twine 4.0?

Reply:
If we add `verbosity` and deprecate `verbose` then we could keep verboes around and use `verbosity` if it's an int and not `None`.

Reply:
We could schedule `verbose` for removal in 4.0

Reply:
Okay, that's helpful roadmap, but it still makes me wonder if supporting `-vv` and `-vvv` at this point provides enough value to Twine's users to warrant the maintenance overhead.

In my opinion, it doesn't, and I'd like to stick with one `-v` that remains a `bool`, unless @sigmavirus24 and @di feel strongly otherwise.